plugins/rikki/heroeslounge/classes/matchmaking/Swiss.php

```diff
@@ -21,6 +21,12 @@ public function createMatches($pairings, $div) {
         $timeoffset = ($pairings[0][0]->region_id - 1)*36000;
         $schedule_date = date('Y-m-d H:i:s', strtotime('next sunday 21:55')+$timeoffset);
         $tbp = date('Y-m-d H:i:s', strtotime('+1 weeks sunday 21:55')+$timeoffset);
+        
+        // if it's the last round, games must be played until next friday (+5 days)
+        if ($div->season->current_round == 10) {
```

Comment:
Any reason you're not using the `$currentRound` property used throughout this class?

Additionally, this would blow up when a season does not have 10 rounds.

Reply:
the createMatch method used it neither, so I assumed it's not initialized at that point. 

Actually good point, this would screw everything up. But: Has there ever be a season with more than 10 rounds?

Reply:
There are seasons with != 10 rounds, e.g. Nexus Rumble.
You can compare to season->round_length which has the number of rounds in the season